lib/tasks/reports.rake

```diff
@@ -7,8 +7,11 @@ namespace :reports do
     report = Report.find(report_id)
     uploader = User.find(requester_id)
 
-    abort "Report #{report.id} has not been approved" unless report.approved?
     abort "Report #{report.id} not found" if report.nil?
```

Comment:
Me again, it should probably be `abort "Report #{report_id} not found" if report.nil?` or we'll either get `4` (the id of nil), or an unexpected error if Rails got wiser about it 😅

Reply:
🤦